lib/sles4sap/ipaddr2.pm

```diff
@@ -119,6 +119,9 @@ Create a deployment in Azure designed for this specific test.
                       during the deployment so before any other part of the test can add
                       additional repo to test.
 
+=item B<trusted_launch> - Enable or disable Trusted Launch. Default 1: Enabled. If configured to 0 
+                          result in az vm create to be executed with '--security-type Standard'
```

Comment:
I think the second part of conditional should be in future tense:
```
If configured to 0 the result in az vm create will be executed with '--security-type Standard'
```

And I think this should work as well:

```
If configured to 0 the result in az vm create is executed with '--security-type Standard'
```